pynest/nest/lib/hl_api_simulation.py

```diff
@@ -190,99 +190,134 @@ def SetKernelStatus(params):
 
     Other parameters
```

Comment:
"Other parameters" doesn't make sense here, since what follows is an explanation of the contents of the `params` dictionary.

Reply:
Would you prefer to have this heading be `Params dictionary`, or something else? I saw that other parameters was used is other pynest docs and stuck to it.

Reply:
I agree with @heplesser regarding ``Other parameters`` and suggest changing it to``Params dictionary`` or ``Parameter dictionary``.